Title:	Research
Slug:	research
Research:	true

<h1 class="top-header">Research</h1>
<ol>
	<li><a href="#researchinterest">Research interest</a></li>
	<li><a href="#pastprojects">Past projects</a>
		<ul>
			<li><a href="#phdproject">PhD project</a>
				<ul>
					<li><a href="#background">Background</a></li>
					<li><a href="#aim">Aim</a></li>
					<li><a href="#expstrategy">Experimental strategy</a></li>
				</ul>
			</li>
			<li><a href="#bachelorthesis">Bachelor thesis project</a></li>
			<!-- <li><a href="#masterthesis">Master thesis</a></li> -->
		</ul>
	</li>
</ol>

## <a id="researchinterest"></a>Research interest &raquo; ##
The development of an embryo involves a series of spatio-temporally ordered cell fate assignations. These decisions are controlled by *mechanical* and *chemical inputs* and lead to the emergence of different tissues and organs. From the level of tissues these decisions appear to be deterministic giving rise to highly robust and reproducible patterns. However, on the single-cell level gene and protein regulatory processes, which underlie cell fate decisions, are subject to stochastic effects which are inherent in biochemical reactions. Understanding how this macroscopic order on a multi-cellular scale arises from stochastic processes on a molecular and cellular level is poorly understood.

> I am interested in how cells make decisions about their fate during development. Particularly, my current research focuses on how cells interpret various environmental signals and integrate them to make precise and reproducible decisions. Furthermore I am interested to which extent this interpretation depends on prior experience (i.e. the internal state of a cell) versus the spatio-temporal dynamics and the statistics of those external cues.

Studying general design principles of core circuitries controlling cell fate decisions as well as elucidating how cells integrate and process information about their environment (including neighboring cells) to generate specific responses might help to understand how organisms go from stochasticity at the molecular and cellular level to deterministic behaviors on the tissue and organismal level.

***

## <a id="pastprojects"></a>Past projects &raquo; ##

### <a id="phdproject"></a>PhD project ###
#### <a id="background"></a>Background
The respiratory lineage in mammals – including the trachea and lung – is specified in the ventral foregut about a day after the endoderm forms during the process of gastrulation. Trachea and lungs of the respiratory lineage consist of an endoderm-derived epithelial sheet surrounded by mesoderm-derived mesenchymal cells. Studies on respiratory and other organ primordia in the foregut such as liver, pancreas, esophagus, and stomach have shown that reciprocal interactions between these two cell lineages are critical for patterning and development of these organs.

Previous work has shown that cell fate specification into respiratory, hepatic, and pancreatic lineages is controlled by external cues from surrounding mesenchyme. Several studies suggested a dosage-dependent role for FGF signaling in this process (Jung et al., 1999[^Jung]; Serls et al., 2005[^Serls]; Calmont et al., 2006[^Calmont]; Wandzioch and Zaret, 2009[^Wandzioch]). Furthermore, canonical Wnt signaling has been proposed to be critical for either the specification or maintenance of the respiratory lineage, respectively (Goss et al., 2009[^Goss]; Harris-Johnson et al., 2009[^Harris-Johnson]).

However, the functional relationship, exact role of these signaling pathways in respiratory lineage specification as well as the temporal sequence of specification from a multipotent endodermal progenitor to respiratory progenitor cell is only poorly understood. Importantly, how each cell senses, interprets and integrates those different signaling cues to make a precise and reproducible cell fate decision has not been addressed so far in this system and is arguably one of the most fundamental questions in current developmental biology and stem cell research.

#### <a id="aim"></a>Aim ###
The overall goal of my PhD project is to better understand how progenitor cells select a certain cell lineage by interpreting and integrating various external chemical cues. I am using the respiratory lineage specification in the mammalian anterior foregut endoderm as a paradigm.

<img class="img-thumbnail" alt="/images/specification-scheme.jpg" src="/images/specification-scheme.jpg">

#### <a id="expstrategy"></a></a>Experimental strategy ###
I am using a combination of *in vitro* chemical genetics in embryo cultures, *in vivo* spatio-temporally controllable genetic strategies as well as live imaging of lineage- and signaling pathway-specific reporters to obtain single-cell and, if possible, time-resolved data to analyze how signals from the FGF, BMP and Wnt/β-catenin pathways are integrated and computed on a cellular and population level to specify respiratory progenitors. These quantitative data will be the basis for mathematical models with which I hope to gain mechanistic insights into this process.

<img class="img-thumbnail" alt="/images/half-embryo.jpg" src="/images/half-embryo.jpg">

***

### <a id="bachelorthesis"></a>Bachelor thesis project ###

High-throughput loss-of-function screening allows a systematic and large-scale quantitative analysis and functional annotation of all genes in the genome. With homogeneous and pathway-specific reporter assays only giving very limited information about cellular phenotypes, high-content microscopy combined with image analysis provides a multi-dimensional and quantitative single-cell readout of loss-of-function phenotypes. 

For my Bachelor thesis project I focused on functionally analyzing a set of previously unknown genes that clustered into a functional module that was highly enriched in genes involved in regulation of genomic integrity and DNA damage response. Based on phenotype similarity we hypothesized that those genes might be involved in similar functions than known genes. I confirmed this hypothesis by using high-throughput in situ cytometry after knockdown of candidate genes. To compare putative conserved functions of those candidate genes I performed similar experiments using Drosophila cultured cells after depletion of homologous genes. Furthermore, because of their predicted function in DNA damage response I also checked for ectopic activation of DDR following knockdown in combination with DNA damage inducing agents.

In summary, we could show that quantitative automated analysis of perturbation phenotypes on a genome-wide scale provides an effective way of annotating unknown genes by categorizing them into known functional modules based on their phenotypic profile.

Part of the work I performed during my Bachelor thesis was later published as part of a paper by Fuchs et al. 2010[^Fuchs] entitled "Clustering phenotype populations by genome-wide RNAi and multiparametric imaging". I have also made my thesis available via figshare[^bscthesis].

<!-- ### <a id="masterthesis"></a>Master thesis ## -->

[^bscthesis]: <b><u>Budjan, C.</b></u> (2008). Functional characterization of novel regulators required for genomic integrity. <i>Bachelor thesis, figshare</i>. doi: <u>10.6084/m9.figshare.1000734</u>.
<a href="http://dx.doi.org/10.6084/m9.figshare.1000734"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank">
  <i class="fa fa-download color-orange"></i>
</a>
[^Calmont]: **Calmont, A., Wandzioch, E., Tremblay, K. D., Minowada, G., Kaestner, K. H., Martin, G. R. and Zaret, K. S.** (2006). An FGF response pathway that mediates hepatic gene induction in embryonic endoderm cells. *Dev Cell* **11**, 339-348. doi: <u>10.1016/j.devcel.2006.06.015</u>.&nbsp;<a href="http://dx.doi.org/10.1016/j.devcel.2006.06.015"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank"><i class="fa fa-download color-orange"></i></a>
[^Fuchs]: **Fuchs, F., Pau, G., Kranz, D., Sklyar, O., Budjan, C., Steinbrink, S., Horn, T., Pedal, A., Huber, W. and Boutros, M.** (2010). Clustering phenotype populations by genome-wide RNAi and multiparametric imaging. *Mol Syst Biol* **6**, 370. doi: <u>10.1038/msb.2010.25</u>.&nbsp;<a href="http://dx.doi.org/10.1038/msb.2010.25"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank"><i class="fa fa-download color-orange"></i></a>
[^Goss]: **Goss, A. M., Tian, Y., Tsukiyama, T., Cohen, E. D., Zhou, D., Lu, M. M., Yamaguchi, T. P. and Morrisey, E. E.** (2009). Wnt2/2b and beta-catenin signaling are necessary and sufficient to specify lung progenitors in the foregut. *Dev Cell* **17**, 290-298. doi: <u>10.1016/j.devcel.2009.06.005</u>.&nbsp;<a href="http://dx.doi.org/10.1016/j.devcel.2009.06.005"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank"><i class="fa fa-download color-orange"></i></a>
[^Harris-Johnson]: **Harris-Johnson, K. S., Domyan, E. T., Vezina, C. M., Sun, X.** (2009). beta-Catenin promotes respiratory progenitor identity in mouse foregut. *Proc Natl Acad Sci USA* **106**, 16287-16292. doi: <u>10.1073/pnas.0902274106</u>.&nbsp;<a href="http://dx.doi.org/10.1073/pnas.0902274106"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank"><i class="fa fa-download color-orange"></i></a>
[^Jung]: **Jung, J., Zheng, M., Goldfarb, M. and Zaret, K. S.** (1999). Initiation of mammalian liver development from endoderm by fibroblast growth factors. *Science* **284**, 1998-2003. doi: <u>10.1126/science.284.5422.1998</u>.&nbsp;<a href="http://dx.doi.org/10.1126/science.284.5422.1998"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank"><i class="fa fa-download color-orange"></i></a>
[^Serls]: **Serls, A. E., Doherty, S., Parvatiyar, P., Wells, J. M. and Deutsch, G. H.** (2005). Different thresholds of fibroblast growth factors pattern the ventral foregut into liver and lung. *Development* **132**, 35-47. doi: <u>10.1242/dev.01570</u>.&nbsp;<a href="http://dx.doi.org/10.1242/dev.01570"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank"><i class="fa fa-download color-orange"></i></a>
[^Wandzioch]: **Wandzioch, E. and Zaret, K. S.** (2009). Dynamic signaling network for the specification of embryonic pancreas and liver progenitors. *Science* **324**, 1707-1710. doi: <u>10.1126/science.1174497</u>.&nbsp;<a href="http://dx.doi.org/10.1126/science.1174497"
    rel="stylesheet"
    title="Download&nbsp;Article"
    target="_blank"><i class="fa fa-download color-orange"></i></a>
